spec/support/features/steps/cases/edit_steps.rb

```diff
@@ -60,6 +63,13 @@ def edit_foi_case_closure_step(kase:, # rubocop:disable Metrics/MethodLength, Me
     .to eq kase.date_responded.month.to_s
   expect(cases_edit_closure_page.date_responded_year.value)
     .to eq kase.date_responded.year.to_s
+  if kase.responded_late?
+    expect(cases_edit_closure_page.get_late_team(kase.late_team_id)).to be_checked
+    expect(kase.late_team_id).not_to eq(late_team_id)
+    if late_team_id
```

Comment:
When would `late_team_id` be false and why?

Reply:
it's an optional parameter